template/functions.go

```diff
@@ -162,16 +162,16 @@ func elapsedTime(printer *locale.Printer, tz string, t time.Time) string {
 		return printer.Plural("time_elapsed.hours", hours, hours)
 	case d == 1:
 		return printer.Printf("time_elapsed.yesterday")
-	case d < 7:
+	case d < 21:
 		return printer.Plural("time_elapsed.days", d, d)
```

Comment:
Hello,

Can you change the `d` printed to days? To keep the same logic, please :)

Reply:
Sorry, I don't understand this comment. Feel free to close this PR if this isn't a welcome change.

Reply:
I apologize for my english.

I like this PR, my point was simply to replace this:
```
return printer.Plural("time_elapsed.days", d, d)
```
by
```
return printer.Plural("time_elapsed.days", d, days)
```

You got my point?